cluster/config-defaults.yaml

```diff
@@ -117,10 +117,12 @@ zmon_worker_cpu: "750m"
 zmon_worker_count: "16"
 {{end}}
 logging_watcher_mem: "200Mi"
-logging_scalyr_mem: "200Mi"
+logging_scalyr_mem: "190Mi"
+logging_slo_heartbeat_mem: "10Mi"
```

Comment:
The minimum currently is `25Mi`, so this will be an effective increase of `15Mi`. Please check if it could be rolled out to existing clusters.

Reply:
Works with these settings in playground.
Btw.: Is 25Mi the limit for pods or containers? (I was assuming the former)

Reply:
It works, but the container will be started with a 25Mi request/limit. This might be too much in some of the clusters, rendering user pods unschedulable. The minimum request is per-container since pods don't have memory requests/limits.